### DISCUSSION TITLE
Nested Blueprints

### DISCUSSION THREAD

**Discussion by Chris530906** [QualiTorque/qualitorque.github.io#149] [Documentation requests] [answered]
Please add documentation for creating and using nested blueprints. e.g.

grains:
  ssm-spec2-nokey:
    kind: blueprint
    spec:
      source: 
        store: test
        path: blueprints

**Comment by TomerAdmon** [comment, marked as answer]
Hi @Chris530906, 
Thanks for raising this in the community!  🙏🙏
We will work on it and update the docs soon 
@Shnitselon

**Comment by Shnitselon** [comment]
Thanks @Chris530906 for the feedback,

Happy to update that we've added a section to the [Blueprint Quickstart Guide](https://docs.qtorque.io/blueprint-designer-guide/blueprint-quickstart-guide) that walks you through how to nest a blueprint within another blueprint.
